msg/DetectedObjsArray.msg

```diff
@@ -0,0 +1 @@
+ObjDetected[] objects
```

Comment:
Can we unify the naming of DetectedObjsArray? So that we have DetectedObjsArray which contains DetectedObjs, or have ObjsDetectedArray which contains ObjsDetected?

Reply:
The convention selected is DetectedObj and DetectedObjsArray